_pkgdown.yml

```diff
@@ -61,7 +61,7 @@ navbar:
   components:
     getstarted:
       text: Get Started
-      href: articles/admiraltemplate.html
+      href: articles/admiralmetabolic.html
     reference:
       text: Reference
       href: reference/index.html
```

Comment:
I think either you should remove the history icon or add the website-versions article.

Reply:
done